lex_node/launch/lex_node.launch

```diff
@@ -3,8 +3,9 @@
     <arg name="node_name" default="lex_node" />
     <!-- If a config file argument is provided by the caller then we will load it into the node's namespace -->
     <arg name="config_file" default="" />
+    <arg name="output" default="log" />
```

Comment:
What are the valid values besides `log`? Are these documented somewhere?

Reply:
Documented here http://wiki.ros.org/roslaunch/XML/node

Reply:
Thanks, I had missed that `output` was part of the standard `<node>` xml arguments. 

Bit of a nitpick, but it would be nice to add a comment documenting what the arg is used for like the other arguments in this file.